packages/types/lib/IStatistic.ts

```diff
@@ -0,0 +1,49 @@
+import type { IActionContextKey } from './IActionContext';
+
+/**
+ * Base interface for statistic trackers that defines all method a statistic should have.
+ */
+export interface IStatisticBase<T> {
```

Comment:
Let's move this interface and the next one to their own dedicated files, corresponding to the interface name.